src/app/(auth)/forgot-password/page.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import TextInput from '@/components/TextInput/index';
+import { H1, H4 } from '@/styles/text';
+// import supabase from '@/api/supabase/createClient';
+// import COLORS from '@/styles/colors';
+import { OuterDiv, FormDiv } from '@/app/(auth)/styles';
+import BigButton from '@/components/BigButton';
+
+export default function ForgotPassword() {
+  const [email, setEmail] = useState('');
+  const { push } = useRouter();
+
+  return (
+    <OuterDiv>
+      <FormDiv>
+        <H1>Forgot Password</H1>
+        <TextInput
+          label="Email"
+          placeholder="email@example.com"
+          type="email"
+          id="email"
+          value={email}
+          setValue={setEmail}
+        />
+        <BigButton type="button" onClick={() => push('/reset-password')}>
```

Comment:
Let's try to work the reset password email into this PR? Can you make it so that the button, when pressed, triggers:

```
const { data, error } = await supabase.auth.resetPasswordForEmail(email, {
  redirectTo: 'http://localhost:3000/reset-password',
})
```
instead of just routing to `/reset-password`?